wgsl/index.bs

```diff
@@ -8178,6 +8146,55 @@ Here are some examples showing this analysis in action:
 </div>
 The same example would also be invalid for the same reason if `continue` was replaced by `break`.
 
+# Assertions # {#assertions}
+
+An <dfn noexport>assertion</dfn> is a check that ensures a boolean condition is satisfied.
+
+<pre class=include>
+path: syntax/global_assert.syntax.bs.include
+</pre>
+
+WGSL defines one kind of [=assertion=]: the const assertion. A const assertion verifies a bool-typed
+const-expression, evaluated at shader-creation time. If that expression evaluates to false, a
+shader-creation error occurs; otherwise, there is no runtime effect. A const assertion can appear at
+module scope or as a statement (see [[#const-assert-statement]]).
```

Comment:
Since the const assertion is now a subsection, I'd suggest delegating the description to there.

```suggestion
WGSL defines one kind of [=assertion=]: the [[#const-assert-statement|const assertion]]. 
```